Sources/SwiftUIBackports/Shared/StateObject/StateObject.swift

```diff
@@ -0,0 +1,111 @@
+import Combine
+import SwiftUI
+
+extension Backport where Content == Any {
+
+    /// A property wrapper type that instantiates an observable object.
```

Comment:
The documentation below is great but in a real use case, the user will type out the `Type` and then see header docs related to that type, rather than the `init`. Can we copy the original docs (modified as required) from Apple's implementation?